app/models/guidance.rb

```diff
@@ -81,7 +81,7 @@ class Guidance < ActiveRecord::Base
 
   # Returns whether or not a given user can view a given guidance
   # we define guidances viewable to a user by those owned by a guidance group:
-  #   owned by the managing curation center
+  #   owned by the default orgs
```

Comment:
Thanks, this should make things a bit easier for people who pick up the codebase but aren't a `curation center`